### DISCUSSION TITLE
Eular angles in updated PyTom good for Relion

### DISCUSSION THREAD

**Discussion by StephenC07** [SBC-Utrecht/pytom-match-pick#155] [Questions]

**Comment by StephenC07** [comment]
Hi there. In the star file generated from pytom_extract_candidates.py are the Euler angles in ZYZ format?  Is there anything else that needs to done to get the right formatting for Relion? Thanks!

**Comment by McHaillet** [comment]
Hi @StephenC07!

Yep, the angles are in the correct format for Relion/Warp. You might need to update the star column prefix though ('ptm' -> 'rln'), see the documentation here: https://github.com/SBC-Utrecht/pytom-match-pick/wiki/merging-annotations

**Comment by StephenC07** [comment]
Thank you so much. I another question. In the star file there are ptmLCCmax, ptmCutOff, ptmSearchStd. Are these parameters needed for an averaging program, other than Relion? Also, to scale the coordinates to unbinned values is it right that rlnDetectorPixelSize in the star file (binned value) will be recognised by Relion so there is no need to alter these values manually? Thanks again!

**Comment by McHaillet** [comment]
Relion should be able to read those columns, at least last time I tried it worked. It probably produces a warning in the terminal along the lines of  'unrecognized starfile column', but should parse it without further issues.

Yep, the detectorpixelsize should be internally used by Relion to scale the coordinates. It is important that Relion knows the tomogram dimensions though!
